src/mmvstr.h

```diff
@@ -163,6 +163,43 @@ typedef char* vstring;
  * construction of a final \ref vstring, it is allocated on the heap as usual.
  *
  * If returned by a function, it is already pushed on the \ref tempAllocStack.
+ *
+ * A `temp_vstring` should never be used as the first argument of a `let`.
+ * This code is INCORRECT:
+ *
+ *   temp_vstring foo = left("foobar", 3);
+ *   let(&foo, "bar"); // this will cause a double free
```

Comment:
Not sure if this is my fault or yours, but this code and the other examples have to be indented 4 spaces relative to the surrounding text to appear as a code block:
```suggestion
 *     temp_vstring foo = left("foobar", 3);
 *     let(&foo, "bar"); // this will cause a double free
```